ResourceManager/GitPluginBase.cs

```diff
@@ -7,6 +7,16 @@
 
 namespace ResourceManager
 {
+    [UsedImplicitly]
+    public abstract class GitPluginWithoutSettingsBase : GitPluginBase
+    {
+        public new bool HasSettings => false;
```

Comment:
Why are we shadowing the property? We have full control over the code.

It won't work the way you expect if the code is written against the parent type, e.g.
![image](https://user-images.githubusercontent.com/4403806/65386668-8b3da400-dd47-11e9-80b2-578148f0c168.png)

In general I suggest avoiding shadowing members unless there is no other way.